### DISCUSSION TITLE
nametag instead of genarated names?

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [networked-aframe/networked-aframe#420] [General]
Hello, I notice in the backend when a user is created that it generates a user ID/username. I was curious if it is possible to use the nametag instead of a generated names?

Joseph

**Comment by vincentfretin** [comment]
Hello, this user id on the backend needs to be unique. That's a different thing than a nametag.

**Comment by Therealjosephchrzempiec** [comment]
I understand but being people put there own unique name in the nametag wouldn't that be enough to be unique to put in the generated UID?

**Reply by vincentfretin** [reply]
No, you can have several persons called Vincent in the room...
If you want to use nametag, you can use a player-info component to have this info.
